### PR TITLE
Reinhardtbotha/user profile and settings

### PR DESCRIPTION
**Pull request by ReinhardtBotha**
Please review my HTML (with some JS) for the User Profile and Settings. 

I still need to implement more JS functionality:

1. Show messages based on what permissions are not allowed
2. Update user information
3. Update profile picture
4. Delete account
5. Change password

### CODE REVIEW

File: user-profile-settings/index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      <!-- change label css to look like button -->
+      <label for="changeProfilePhotoBtn">Change profile photo</label>
+      <!-- hide input field in css -->
+      <input
+        id="changeProfilePhotoBtn"
+        type="file"
+        accept="image/jpeg, image/jpg, image/png"
+      />
+
+      <button type="submit" id="saveProfileInfoBtn">Save</button>
```
Comment (whamester) [inline user-profile-settings/index.html:38]: form="profileForm"
https://www.w3schools.com/tags/att_button_form.asp

File: index.html [hunk 2]
```diff
@@ -30,6 +30,7 @@ <h1>PWA Home</h1>
     <span id="username"></span>
```
Comment (whamester) [inline index.html:30]: profile.html or settings.html

File: user-profile-settings/index.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>User Profile & Settings</title>
+
```
Comment (ALFAROSO) [inline user-profile-settings/index.html:7]: I'm not sure but I think  ./  is not necessary just "style.css" and "script.js"

**Review by ALFAROSO — COMMENTED** [review]
Looks good for me

File: user-profile-settings/index.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    <!-- Link to css file -->
+    <link rel="stylesheet" href="./style.css" />
+    <!-- Link to js file -->
+    <script type="module" src="./script.js" defer></script>
```
Comment (ALFAROSO) [inline user-profile-settings/index.html:11]: I'm not sure but I think  ./  is not necessary just "style.css" and "script.js"

Comment (ReinhardtBotha) [reply user-profile-settings/index.html:11]: I think both are fine, maybe we should decide on a standard way of writing it?

File: user-profile-settings/index.html (hunk 1 above)
Comment (ReinhardtBotha) [reply user-profile-settings/index.html:38]: done

File: user-profile-settings/index.html (hunk 4 above)
Comment (whamester) [reply user-profile-settings/index.html]: it's fine.

**Review by whamester — APPROVED** [review]

**Pull request merged**